velox/exec/TaskStructs.h

```diff
@@ -22,7 +22,11 @@ namespace facebook::velox::exec {
 
 class Driver;
 class JoinBridge;
+class LocalExchangeMemoryManager;
```

Comment:
The compiler does not currently complain since TaskStructs.h is only included inside ./velox/exec/Task.h and the headers that define these classes come before this include.